<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiCenterDialog</class>
 <widget class="QDialog" name="multiCenterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>411</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>619</width>
      <height>409</height>
     </rect>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>109</width>
        <height>409</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_Install">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>安装方式</string>
       </property>
       <property name="icon">
        <iconset resource="../qfi.qrc">
         <normaloff>:/qfi/images/set.png</normaloff>:/qfi/images/set.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>机架选择</string>
       </property>
       <property name="icon">
        <iconset resource="../qfi.qrc">
         <normaloff>:/images/X型四旋翼.PNG</normaloff>:/images/X型四旋翼.PNG</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_RCCfg">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>100</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>RC配置</string>
       </property>
       <property name="icon">
        <iconset resource="../qfi.qrc">
         <normaloff>:/qfi/images/remote1.png</normaloff>:/qfi/images/remote1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_AutoCfg">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>自驾参数配置</string>
       </property>
       <property name="icon">
        <iconset resource="../qfi.qrc">
         <normaloff>:/qfi/images/airplane.png</normaloff>:/qfi/images/airplane.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_controlset">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>260</y>
         <width>101</width>
         <height>38</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>控制律配置</string>
       </property>
       <property name="icon">
        <iconset resource="../qfi.qrc">
         <normaloff>:/qfi/images/ask.png</normaloff>:/qfi/images/ask.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_analogset">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>101</width>
         <height>38</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>模拟量标定</string>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_HMCSet">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>101</width>
         <height>38</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>磁力计校准</string>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_slashset">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>360</y>
         <width>101</width>
         <height>38</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>210</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>喷洒系统配置</string>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>501</width>
       <height>361</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qfi.qrc"/>
 </resources>
 <connections/>
</ui>
